LANGS.md

```diff
@@ -1,4 +1,4 @@
 * [English](en/)
-* [Chinese (中文)](zh/)
+<!-- * [Chinese (中文)](zh/)
```

Comment:
Undo. This change turns off our translation for Chinese and Korean on the live site. I sometimes do this to reduce build times locally, but you shouldn't save it to master.